locales/en-US/strings.properties

```diff
@@ -117,3 +117,11 @@ pocket_feedback_header=The best of the web, curated by over 25 million people.
 # (pocket_feedback_header) to provide more information about Pocket.
 pocket_feedback_body=Pocket, a part of the Mozilla family, will help connect you to high-quality content that you may not have found otherwise.
 pocket_send_feedback=Send Feedback
+
+# LOCALIZATION NOTE (migrate_*): This is shown in the migration message
+migrate_manual_import_msg=No problem. We can start you off with a few popular sites instead. You can always
+# LOCALIZATION NOTE (migrate_manual_import_link_msg): This is shown as the link in message
+migrate_manual_import_link_msg=import your information
+migrate_manual_import_msg_trail=later.
```

Comment:
Ehh. I could see some locales wanting to just put "." in the trailing text, so if there's an implicit space to join, it would appear as "Stuff stuff later [link] ."